helpers/command.go

```diff
@@ -28,10 +30,27 @@ func RunCommand(command *exec.Cmd) error {
 	command.Stderr = os.Stderr
 	command.Stdin = os.Stdin
 	logrus.Debugf("running command %s with args %v", command.Path, command.Args)
-	if err := command.Run(); err != nil {
-		logrus.Debug("last command was not successful")
+
+	if err := command.Start(); err != nil {
+		logrus.Error("unable to start command")
 		return err
 	}
+
+	if err := command.Wait(); err != nil {
```

Comment:
`command.Run` also returns `*ExitError`: https://golang.org/pkg/os/exec/#Cmd.Run, why not just use that?

Reply:
so we don't try to extract exit code in the case that `Start()` returned an error